commons/src/main/java/org/dhis2/commons/date/DateUtils.java

```diff
@@ -0,0 +1,232 @@
+package org.dhis2.commons.date;
+
+import androidx.annotation.NonNull;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+public class DateUtils {
+
```

Comment:
Should we migrate this class to kotlin now that it in the commons module? Not in this issue but in a new one? What do you think?

Reply:
Yes I agree, with the DateUtils I think we should create a new issue to move all the class methods, and It would be great to migrate it to kotlin